### PR TITLE
Update bootstrap script for xenial support

### PR DESCRIPTION
**Pull request by humblearner**
Generic bootstrap script for trusty and xenial.
Test:
- [x] trusty
- [x] xenial

### CODE REVIEW

File: scripts/st2bootstrap-deb.sh [hunk 1]
```diff
@@ -14,6 +14,7 @@ ST2WEB_PKG_VERSION=''
 ST2CHATOPS_PKG_VERSION=''
 USERNAME=''
 PASSWORD=''
+SUBTYPE=`lsb_release -a 2>&1 | grep Codename | grep -v "LSB" | awk '{print $2}'`
```
Comment (lakshmi-kannan) [inline scripts/st2bootstrap-deb.sh]: lsb_release isn't always installed. Best to quit with an error saying if SUBTYPE not in [xenial, trusty]

**Comment by armab** [comment]
Just a note to close another PR: https://github.com/StackStorm/st2-packages/pull/327 when this one is implemented.

**Comment by armab** [comment]
Looks like this PR needs also `st2_bootstrap.sh` changes

File: scripts/st2bootstrap-deb.sh [hunk 2]
```diff
@@ -100,18 +104,19 @@ install_st2_dependencies() {
...
+  sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv EA312927
+  echo "deb http://repo.mongodb.org/apt/ubuntu ${SUBTYPE}/mongodb-org/3.2 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.2.list
 
   sudo apt-get update
   sudo apt-get install -y mongodb-org
-}
 
+  if [[ "$SUBTYPE" == 'xenial' ]]; then  
+    sudo systemctl enable mongod
+    sudo service mongod start
```
Comment (bigmstone) [inline scripts/st2bootstrap-deb.sh]: This works, but really should be `sudo systemctl start mongod`

**Pull request merged**